packages/core-extensions/src/nativeFixes/host.ts

```diff
@@ -0,0 +1,56 @@
+import { app, nativeTheme } from "electron";
+
+const config = moonlightHost.getConfig("nativeFixes") ?? {};
+
+const enabledFeatures = app.commandLine
+  .getSwitchValue("enable-features")
+  .split(",");
+
+moonlightHost.events.on("window-created", function (browserWindow) {
+  if (config.devtoolsThemeFix ?? true) {
```

Comment:
This would be `any`, right? We should probably bind the config values with [getConfigOption<T>](https://github.com/moonlight-mod/moonlight/blob/7926793e628d6a7ad044032e8ad4858bde13da22/packages/core-extensions/src/spacepack/webpackModules/spacepack.ts#L207-L211) instead.